app/assets/javascripts/discourse/app/controllers/sidebar-section-form.js

```diff
@@ -325,23 +363,111 @@ export default Controller.extend(ModalFunctionality, {
     return this.model.links.filter((link) => !link._destroy);
   },
 
+  get activeSecondaryLinks() {
+    return this.model.secondaryLinks?.filter((link) => !link._destroy);
+  },
+
   get header() {
     return this.model.id
       ? "sidebar.sections.custom.edit"
       : "sidebar.sections.custom.add";
   },
 
+  @bind
+  reorder(linkIdFrom, linkIdTo, above) {
```

Comment:
a bit complicated as links can be moved between primary segment (model.links) and more segment(model.secondaryLinks)